.changeset/hot-ligers-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@eth-optimism/contracts-ts': minor
```

Comment:
Going minor because the contracts-ts package version is pinned to contracts and we don't want to major bump contracts